wordpress.org/public_html/wp-content/themes/pub/wporg-main/front-page.php

```diff
@@ -27,20 +27,20 @@
 $swag_class = $showcase ? 'col-4' : 'col-2';
 $user_class = $showcase ? 'col-12' : 'col-2';
 
-$wp20_url = 'https://wp20.wordpress.net/';
+$banner_url = 'https://wordpress.tv/2023/10/13/gutenberg-next-with-matt-mullenweg/';
```

Comment:
Sorry for the delay on this. State of the Word is the most current event now, how about linking to that instead?
```suggestion
$banner_url = 'https://wordpress.org/state-of-the-word/';
```